applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js

```diff
@@ -2,43 +2,89 @@
 'require form';
 'require fs';
 'require view';
+'require uci';
+
+function disk(devs, option, section_id) {
+	var v = uci.get('hd-idle', section_id, 'disk') || '';
+	var e = devs.findIndex(function(itm) { return itm.name == v; } );
+	return E('span', e != -1 ? option.map(o => devs[e][o]).filter(o => o).join(' ').trim() : '');
```

Comment:
Please don't use arrow functions (yet), we're still aiming for ES5 syntax compatibility atm.